community/frage-guide.md

```diff
@@ -2,35 +2,22 @@
 tag: frage-guide
 alias: []
 ---
-
-Hey, es scheint als wolle der User hier drüber dir helfen, du hast allerdings deine Frage unvollständig oder falsch formuliert!
-
+Du hast deine Frage anscheinend unvollständig oder falsch formuliert!
 Hier sind ein paar Tipps zum richtigen formulieren einer Frage:
-
-1. Vermeide Metafragen! Was genau das ist kannst du hier nachlesen: http://metafrage.de/
-
-2. Versuche dein Problem genau zu formulieren, beschreibe detailliert:
-      - Was du erreichen willst,
-      - Was du bereits hast,
-      - Woran es scheitert/zu welchen Abschnitt du Hilfe brauchst.
-
-3. Füge deinen bisherigen Fortschritt hinzu, verwende zum Teilen von Quellcode etc. Seiten wie hastebin:
-https://hastebin.com/
-https://hasteb.in/
-https://gist.github.com/
-
--> Wenn du eine gewisse Problemstellung lösen willst, reicht ein Teil deines Quellcodes
--> Wenn du eine Fehlermeldung bekommst, teile am besten die Fehlermeldung und die ganze Klasse (Java, etc.) oder das ganze Dokument
-      -> Fehlermeldungen enthalten meistens Zeilenangaben, diese bringen uns
-           nichts, wenn wir nur teile des Quellcodes erhalten
-      -> Teile Fehlermeldung und Quellcode immer getrennt!
-
+1. Vermeide [Metafragen](<https://metafrage.de/>)!
+2. Versuche dein Problem genau zu formulieren:
+  - Was du erreichen willst
+  - Was du bereits hast
+  - Woran es scheitert/zu welchen Abschnitt du Hilfe brauchst
+3. Füge deinen bisherigen Fortschritt hinzu, verwende zum Teilen von Code Seiten wie [hastebin](<https://hastebin.com/>) oder [Github Gist](<https://gist.github.com/>):
+  - Wenn du eine gewisse Problemstellung lösen willst, reicht ein Teil deines Quellcodes
```

Comment:
```suggestion
  - Wenn du ein spezifisches Problem lösen willst, reicht ein Teil deines Quellcodes
```
Problemstellung klingt irgendwie so, als wäre es ein formell definiertes Problem in der Schule, lol